<ui version="4.0" >
 <class>PositionEditorWidget</class>
 <widget class="QWidget" name="PositionEditorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>72</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_lat" >
       <property name="text" >
        <string>Latitude</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="lat_sign" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QSpinBox" name="lat_degrees" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string> °</string>
       </property>
       <property name="maximum" >
        <number>89</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QDoubleSpinBox" name="lat_minutes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string> '</string>
       </property>
       <property name="maximum" >
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QComboBox" name="lat_NS" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_lon" >
       <property name="text" >
        <string>Longitude</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="lon_sign" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QSpinBox" name="lon_degrees" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string> °</string>
       </property>
       <property name="maximum" >
        <number>359</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QDoubleSpinBox" name="lon_minutes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string> '</string>
       </property>
       <property name="maximum" >
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4" >
      <widget class="QComboBox" name="lon_EW" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
